config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -14,9 +14,15 @@ presubmits:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240515-17c6d50e24-master
         command:
         - runner.sh
-        args:
-        - make
-        - build
+        - |          
+          mkdir temp_build_logs
+          targets=('linux-amd64' 'linux-arm64' 'linux-arm' 'linux-ppc64le' 'linux-s390x' 'linux-386' 'darwin-amd64' 'darwin-ard64' 'windows-amd64')
```

Comment:
We should not store list of supported platforms in separate repo if we can avoid it. 

Instead of having the list here, we should introduce target `make build-all` to build all architectures. 
Would be also nice to have it based on subtargets `build-amd64`, `build-arm64` etc.

Reply:
>  Would be also nice to have it based on subtargets build-amd64, build-arm64 etc.
Yes, I can raise a PR for it.

Reply:
https://github.com/etcd-io/etcd/pull/18083